### PR TITLE
truffle-contract-size

### PR DESCRIPTION
**Pull request by Rishabh42**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -201,7 +201,74 @@ jobs:
...
+          - v1-dependencies-
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "package.json" }}
+          # fallback to using the latest cache if no exact match is found
+          - v1-dependencies-
+
+      - run: cd compound-protocol && yarn install --lock-file
```
Comment (jchittoda) [inline .circleci/config.yml:250]: do we need these commands, as we now have all executed in `npm i` ?

File: .circleci/config.yml [hunk 2]
```diff
@@ -201,7 +201,74 @@ jobs:
...
+  deploy-ready:
+    docker:
+      # specify the version you desire here
+      - image: circleci/node:10.23.0
+      # Specify service dependencies here if necessary
+      # CircleCI maintains a library of pre-built images
+      # documented at https://circleci.com/docs/2.0/circleci-images/
+      # - image: circleci/mongo:3.4.4
+      # - image: trufflesuite/ganache-cli
+      #   command: ganache-cli --gasLimit 20000000 --gasPrice 20000 --defaultBalanceEther 1000000000 --allowUnlimitedContractSize true
```
Comment (jchittoda) [inline .circleci/config.yml:215]: Remove unused commented commands

File: package.json [hunk 3]
```diff
@@ -20,7 +20,8 @@
     "coverage-setup": "node_modules/.bin/testrpc-sc -p 8546 --gasLimit 0x1fffffffffffff --gasPrice 1 --defaultBalanceEther 1000000000 --allowUnlimitedContractSize true",
     "lint": "npx solium -d contracts/",
     "ganache": "ganache-cli --gasLimit 0x1fffffffffffff --gasPrice 20000000 --defaultBalanceEther 1000000000 --allowUnlimitedContractSize true --mnemonic “begin vessel olive rocket pink distance admit foam lizard type fault enjoy”",
-    "buidler-evm": "npx buidler node"
+    "buidler-evm": "npx buidler node",
+    "contract-size": "truffle run contract-size --contracts SavingAccount Accounts Bank InitializablePausable Constant GlobalConfig AccountTokenLib BitmapLib SavingLib Utils ETHPerFIN ETHPerLPToken TokenRegistry"
```
Comment (jchittoda) [inline package.json:24]: Include `xxxxProxies.sol` contracts as well.

**Review by jchittoda — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (Rishabh42) [reply .circleci/config.yml:250]: Yes, we don't require those commands now. 
I had commented them in the `test` job, will remove them from all other jobs as well

File: package.json (hunk 3 above)
Comment (Rishabh42) [reply package.json]: Added those

File: .circleci/config.yml (hunk 2 above)
Comment (Rishabh42) [reply .circleci/config.yml:215]: Removed

**Review by jchittoda — APPROVED** [review]

**Pull request merged**